packages/kotti-ui/source/kotti-field/KtField.vue

```diff
@@ -251,13 +251,11 @@ export default defineComponent<{
 					}
 				}
 
-				&:focus-within {
```

Comment:
Why did this have to change?

Reply:
because the hooks hackily append kotti classes to the el-date inner input fields

since we now are context aware, the date-picker is appended to the parent, and if the specificity is not as high, focusing on the parent field was focusing on all the inner input fields (because of focus-within)

test it out in the browser, by bringing back those classes as they were, and you will see how the behavior is altered